.changeset/cool-days-look.md

```diff
@@ -0,0 +1,7 @@
+---
+'@astrojs/starlight': patch
+---
+
+Make Starlight compaticle with server output mode.
+
+Starlight pages are always prerendered, even when `output: 'server'`.
```

Comment:
Suggesting a couple of small tweaks to match our changelog style (third-person “makes” over imperative “make”) and the Astro docs terminology of “on-demand server rendering”:

```suggestion
Makes Starlight compatible with [on-demand server rendering](https://docs.astro.build/en/guides/server-side-rendering/) (sometimes referred to as server-side rendering or SSR).

Starlight pages are always prerendered, even when using `output: 'server'`.
```